core/tests/protractor.conf.js

```diff
@@ -227,7 +227,7 @@ exports.config = {
   capabilities: {
     browserName: 'chrome',
     chromeOptions: {
-      args: ['--lang=en-EN', '--window-size=1285x1000']
+      args: ['--lang=en-EN', '--window-size=1285x1000', '--use-fake-device-for-media-stream', '--use-fake-ui-for-media-stream']
```

Comment:
Why add these flags?

Reply:
Running test locally, my browser required adding these flags to enable recording audio from microphone.

Reply:
Can you add this as a comment?

Reply:
Sure!